wechat-minip/pages/components/coverpage/index.ts

```diff
@@ -0,0 +1,27 @@
+import { App, EachPage } from '../../../app';
+import Common from '../common';
+
+const app = getApp<App>();
+
+interface Data {
+  pageVisible?: boolean;
+  navbarTitleVisible: boolean;
+}
+
+const initialData: Data = {
+  navbarTitleVisible: false,
+};
+
+Page({
+  ...Common,
+  ...(app.eachPage as EachPage<Data>),
+  data: { ...app.eachPage.data, ...initialData },
+  onLoad(query: Record<string, string>) {
+    this.onLoadOne(query, '叠加页').commit();
+  },
+  setVisible() {
+    const nextPageVisible = !this.data.pageVisible;
+    app.ui.updateNativeContainerStyle(nextPageVisible ? { mode: 'dark' } : {});
```

Comment:
期望在关闭叠加层时恢复导航栏状态，需要置 `undefined` 以取用默认的 `ui.UserSetting` ，使用空对象会自动解析为 `mode: 'auto'`。

```suggestion
    app.ui.updateNativeContainerStyle(nextPageVisible ? { mode: 'dark' } : undefined);
```